tests/next_tests/integration_tests/cases.py

```diff
@@ -75,7 +74,7 @@
 C2E = gtx.FieldOffset("E2V", source=Edge, target=(Cell, C2EDim))
 
 ScalarValue: TypeAlias = np.int32 | np.int64 | np.float32 | np.float64 | np.generic
-FieldValue: TypeAlias = gtx.Field | embedded.LocatedFieldImpl
+FieldValue: TypeAlias = gtx.Field  # | embedded.LocatedFieldImpl
```

Comment:
I would either add an explanation for the commented out code or delete the comment.

Reply:
deleted